rust/item-cpi/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+
+anchor_gen::generate_cpi_crate!("item.json");
+
+anchor_lang::declare_id!("CKAcdJsyzBxHJRHgKVEsVzjX9SNcvut8t3PUD34g7ry4");
```

Comment:
I assume this would be the item program pubkey?

Reply:
yeah, i dont have the keys locally so thats the one i generated